pom.xml

```diff
@@ -333,26 +333,26 @@
             <configuration>
               <relocations>
                 <relocation>
-                  <pattern>com</pattern>
-                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.com</shadedPattern>
+                  <pattern>com.</pattern>
+                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.com.</shadedPattern>
                   <excludes>
                     <exclude>com.google.cloud.spanner.**</exclude>
                   </excludes>
                 </relocation>
                 <relocation>
-                  <pattern>android</pattern>
-                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.android</shadedPattern>
+                  <pattern>android.</pattern>
+                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.android.</shadedPattern>
                 </relocation>
                 <relocation>
-                  <pattern>io</pattern>
-                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.io</shadedPattern>
+                  <pattern>io.</pattern>
+                  <shadedPattern>com.google.cloud.spanner.jdbc.shaded.io.</shadedPattern>
                   <excludes>
-                    <exclude>io.grpc.netty.shaded.**</exclude>
```

Comment:
Would you mind elaborating a little bit on why this change is needed? The PR description mentions 'errors on service provider...'. Could you add the specific error message that you are getting?

Reply:
I couldn't replicate the error. so I've rolled back netty related fix. If I encounter again, I open a separate ticket. Sorry for the mess.

Reply:
No worries. Thanks for the detailed analysis and fix for the shading problem.